### DISCUSSION TITLE
Feature Request: option to for public view without account

### DISCUSSION THREAD

**Discussion by demlak** [go-shiori/shiori#334] [Feature requests]
Would be nice to share the whole database or just specific tags in webserver-view for people without account. 
just view-mode.

**Comment by ghost** [comment]
Seconding this request, it would make possible to use shiori as pinboard / shaarli alternative. See [shaarli demo](https://demo.shaarli.org/) for comparison.

**Comment by zygiss** [comment]
I'm also interested in such functionality, although my use-case is different - I want to run Shiori behind a reverse proxy that handles authentication already, so I want to avoid having to auth twice.

How much work would it be to implement this?

Thanks for awesome work by the way, Shiori is light and fast, and a pleasure to use as a result.

**Comment by fmartingr** [comment]
> Possible duplicate of #331
